Source/XUnitExtensions/AssertWithDiff.cs

```diff
@@ -7,7 +7,7 @@
 namespace XUnitExtensions {
   public class AssertWithDiff {
     public static void Equal(string expected, string actual) {
-      var diff = InlineDiffBuilder.Instance.BuildDiffModel(expected, actual);
+      var diff = InlineDiffBuilder.Instance.BuildDiffModel(expected, actual, false);
```

Comment:
We'll see if the CI confirms this, but I recall needing to ignore whitespace to have this work on all platforms, because otherwise it would complain about `/r/n` not matching `/n`.

Reply:
You mean this is one of those instances where we want to test on all platforms?